salt/_modules/metalk8s_monitoring.py

```diff
@@ -203,7 +203,7 @@ def _requests_alertmanager_api(route, method='GET', **kwargs):
                 "Malformed response returned from Alertmanager API: {!s}: {}"
                 .format(exc, response.text)
             )
-        raise CommandExecutionError(error)
+        raise CommandExecutionError(error)  # pylint: disable=raise-missing-from
```

Comment:
Same?

Reply:
Do we really want to raise the ValueError part of `response.json` ? 
Hmm maybe it make sense so that we know what the problem with the response content as it may not be obvious even if you see the response text
Ok fair I will update this one